### ISSUE TITLE
[TW-1854] verbose=override token

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1868]
_David Patrick on 2016-09-22T23:52:05Z says:_ 

 Many scripts use configuration overrides, and in many of those cases, the “Configuration override rc.defaultwidth:120” message is undesirable. Currently, verbosity control of rc.override messages is part of the verbose=affected token. This RFE is for a new verbose token “override” that is used specifically to enable or disable rc override messages.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2016-09-22T23:52:05Z
Modified: 2017-01-17T01:01:07Z
```

**Comment by tbabej** [comment]
This was implemented in #1947 and released as part of 2.5.2 release.
